plugin/src/main/webapp/js/issues-detail.js

```diff
@@ -1,5 +1,11 @@
-/* global jQuery3, view, echartsJenkinsApi */
+/* global jQuery3, view, echartsJenkinsApi, bootstrap5, trendDefaultStorageId */
```

Comment:
trendDefaultStorageId doesn't seem to be referenced elsewhere.

Reply:
Thanks for spotting!

Reply:
I was searching for "storage" because it was not clear to me whether the configuration is stored in the browser, in the Jenkins project, or in the Jenkins user profile.

Reply:
Currently the configuration is stored in the Browser local storage only. If required this can be changed later on.